serverless/src/requeueOrder/handler.js

```diff
@@ -60,7 +60,7 @@ const requeueOrder = async (event, context) => {
     } else if (type === 'ECHO ORDERS') {
       // Submits to Legacy Services (CMR) and is often referred to as an
```

Comment:
Is this comment invalid now since legacy-services is no longer getting called for this